reference/channels.v3.yml

```
@@ -2225,6 +2341,61 @@ components:
           description: |
             The total number of pages in the collection.
       x-internal: false
+    Site:
+      type: object
+      properties:
+        id:
+          type: integer
+        url:
+          type: string
+          description: The Fully Qualified URL (including host and scheme) where this site is hosted. All URLs generated for this site will be appended to this.
+          example: 'http://kittens.mybigcommerce.com/'
+        channel_id:
+          type: integer
+          description: 'The channel this site is attached to. Each site belongs to a single channel, and each channel can have either zero or one sites.'
+        created_at:
+          type: string
+          description: When was this site created? RFC 3339
```

For these descriptions that low-key have the datatype of a standard, please revise into a sentence that reflects that connection. Also, dashes. and maybe links. I'm not 100% that `datetime` follows the style guide, but you get the idea. Do a search through for these.

```suggestion
          description: The datetime that this site was created, formatted as an [RFC-3339](https://www.ietf.org/rfc/rfc3339.txt) string.
```